.github/workflows/refund-nightly-skyline.yml

```
@@ -0,0 +1,39 @@
+---
+name: Refund Nightly Local Tests
+on: # yamllint disable-line rule:truthy
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  refund_ci:
+    name: Refund CI
+    uses: ./.github/workflows/apex-ci.yml
+    with:
+      lint: true
+      unit_test: true
+      e2e_apex_test: true
+      e2e_skyline_test: false
```

```suggestion
      e2e_skyline_test: true
```
we need to activate `E2E Skyline Tests`